microcosm_eventsource/stores/rollup.py

```diff
@@ -88,6 +88,12 @@ def search(self, **kwargs):
             for row in self._search_query(aggregate, **kwargs).all()
         ]
 
+    def search_first(self, **kwargs):
+        results = self.search(limit=1, **kwargs)
```

Comment:
I'd probably remove the `limit`. It doesn't have an effective difference, and it may already be specified in `kwargs`, which will cause it to error out. Just select the first item that's returned.